apps/user/models/team.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type Team struct {
+	gorm.Model
+	Name        string `gorm:"type:varchar(30);not null"`
```

Comment:
We should have a varchar of 64 length as the team name could be longer.

Reply:
done